device/chip/local_chip.cpp

```diff
@@ -6,9 +6,14 @@
 
 #include "umd/device/chip/local_chip.h"
 
+#include "umd/device/tt_device/tt_device.h"
+
 namespace tt::umd {
 
-LocalChip::LocalChip(tt_SocDescriptor soc_descriptor) : Chip(soc_descriptor) {}
+LocalChip::LocalChip(tt_SocDescriptor soc_descriptor, int pci_device_num) :
+    Chip(soc_descriptor), tt_device_(TTDevice::create(pci_device_num)) {}
```

Comment:
nit: `pci_device_num` might sound like total number of pci devices. Consider renaming this to something like `pci_device_id`